_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+
+# Some other neat tips
+Here are a couple of neat Pebble tips that I've seen recently, that maybe you didn't already know:   
+
+   - When you start playing music on your phone, the stock music app will move to the top of your app list on your watch, making it easy to open by pressing select twice from your watchface   
+   - Holding back from a notification will toggle quiet time
+   - Enabling 'Stand-By Mode' from Settings -> System on your watch will disable bluetooth when your Pebble doesn't move for 30 minutes, saving battery power   
+
+# Darksky API Shutdown
+You might also have seen recently that Apple have purchased Darksky, one of the most popular weather API providers for Pebble apps and faces. Apple are going to shut down the API June 2021, and have already stopped issuing new API keys. Whilst this won't affect the Rebble weather service, watchfaces that require this API will no longer work properly. There's plenty of time to come up with a replacement solution though, so I'm hopeful.   
+
+# What else are Rebble up to?
+Work continues in the background by our [trusty band of volunteers](https://rebble.io/team/), which can be followed on [Github](https://github.com/Pebble-dev/) and [Discord](https://rebble.io/Discord).
+
+   - RebbleOS (the replacement firmware) continues to get frequent updates, with automated testing being a recent addition, as well as loading apps over bluetooth! Check out [#firmware](https://rebble.io/Discord) for more.
+   - The RWS infrastructure has seen some upgrades recently, moving some services to Google Cloud and integrating operational analytics and observability tooling to make finding and fixing errors and choke points easier.
+   - RWS Bugfixes! - For those of you on iOS, a fix was recently deployed that resolves the locker sync issue some of you were experiencing. If this has affected you, [check out this Reddit post](https://old.Reddit.com/r/Pebble/comments/fueo35/potential_fix_for_ios_rebble_app_store_issues_try/). You'll also notice that the 'see all' buttons that mysteriously disappeared from the appstore [are back for good now](https://i.imgur.com/T7sR8WG.png).
+   - Work on the [developer portal](https://rebble.io/submit) has begun, to make submitting new apps easier.
+   - There's even some work on a replacement mobile app, though this is a *huge* piece of work likely to take time.
+
+So there you have it, the answer to some of the more commonly asked questions and hopefully a demonstration of just how much functionality you can get with a few apps and some timeline trickery! At some point I'd like to do a blog post of my Pebble workflow, and all the cool things I use it for, so *watch* 😉 this space.
+
+Finally, if you have a question that wasn't covered in this post, check the [Common Issues section](https://rebble.io/howto/#common-issues) or stop by #rebble-help [ on Discord](https://rebble.io/Discord). If there's one thing the Pebble community isn't short on, it's people willing to help!
```

Comment:
again, Discord links are bad? :link: :boom:

Reply:
Find and replace to fix capitalisation is not my friend :(